cmd/podman/main.go

```diff
@@ -52,14 +52,14 @@ func parseCommands() *cobra.Command {
 		// Command cannot be run rootless
 		_, found := c.Command.Annotations[registry.UnshareNSRequired]
 		if found {
-			if rootless.IsRootless() && os.Getuid() != 0 {
+			if rootless.IsRootless() && os.Getuid() != 0 && c.Command.Name() != "scp" {
```

Comment:
Just remove the `Annotations` when creating the command https://github.com/containers/podman/pull/12429/files#diff-444094a8356a4130e21870b7a63e7c6d60cf4554a7b33bc9f01beafeb4eefb74R34-R35.

Reply:
This is necessary to allow scp to be run rootless. These annotations were designed (it seems) to only be allowed when run through sudo or as root directly, scp needs to be run as a user. I remove the annotations when we run, but this allows the annotations to be used.